posts/2016-02-14-serverless-v1.7.0.md

```diff
@@ -1,6 +1,6 @@
 ---
-title: CloudWatch Events support, deprecation notices, lower memory usage during zipping, bugfixes and improvements in Serverless Framework v1.7
-description: CloudWatch Events support, deprecation notices, lower memory usage during zipping, bugfixes and improvements in the Serverless Framework v1.7 release.
+title: CloudWatch Events support, CLI deprecation warnings, lower memory usage during zipping, bugfixes and improvements in Serverless Framework v1.7
```

Comment:
Title is super long. 

You want it to be around 60-70 characters long.

Also, try and make it catchy and focus on the main feature and what kinds of super powers it might unleash for users.

Like: 
`Trigger Lambda functions with CloudWatch Events with Serverless v1.7`
`React to AWS infrastructure events with CloudWatch Events + Lambda w/ Serverless v1.7`